sccm/apps/deploy-use/create-applications.md

```diff
@@ -330,6 +330,9 @@ Continue to the next section on using a custom script as a detection method. Or
 
 Configuration Manager checks the results from the script. It reads the values written by the script to the standard output (STDOUT) stream, the standard error (STDERR) stream, and the exit code. If the script exits with a non-zero value, the script fails, and the application detection status is *Unknown*. If the exit code is zero, and STDOUT has data, the application detection status is *Installed*.
 
+> [!TIP]
+> This may not be intuitive for new Configuration Manager administrators.  When writing your detection script you cannot return any output unless you want to return that the application is installed.  See examples below the tables for help.
```

Comment:
I suggest rewording a bit for style, let me know if it still means the same as your intent 😄 

```suggestion
> When writing a detection script, you can only return output (data in STDOUT) when it returns a zero exit code. The zero exit code means it detects the application as installed. For more information, see the following examples.  
```

Reply:
Hi @aczechowski, still feels a bit confusing to me, how about somethinglike:

> When writing a detection script, if you return a zero exit code but don't return output (data in STDOUT), the application will not be detected as installed.  For more information, see the following examples.

Reply:
That looks great, thanks @ChrisKibble (sorry for the slow response, been busy post-MMS with 1905 tech preview! 😮 💥 )

Reply:
Awesome, change made in patch-15!

Reply:
And good luck with 1905 👍